supertokens_python/recipe/emailverification/emaildelivery/service/backward_compatibility/__init__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from os import environ
+from typing import Any, Awaitable, Callable, Dict, Union
+
+from httpx import AsyncClient
+from supertokens_python.ingredients.emaildelivery.types import \
+    EmailDeliveryInterface
+from supertokens_python.recipe.emailverification.interfaces import \
+    TypeEmailVerificationEmailDeliveryInput
+from supertokens_python.recipe.emailverification.types import User
+from supertokens_python.supertokens import AppInfo
+
+
+def default_create_and_send_custom_email(app_info: AppInfo) -> Callable[[User, str, Dict[str, Any]], Awaitable[None]]:
+    async def func(user: User, email_verification_url: str, _: Dict[str, Any]):
+        if ('SUPERTOKENS_ENV' in environ) and (environ['SUPERTOKENS_ENV'] == 'testing'):
+            return
```

Comment:
need to add test that it comes in here

Reply:
Added tests in `tests/emailpassword/test_emaildelivery.py`